### ISSUE TITLE
error at download

### ISSUE THREAD

**Issue by josserman** [Esgrove/recordpool-dl#35]
Finally made some progress, but now I'm stuck at this point. Any suggestions? 

Starting ChromeDriver...

pool: Beatjunkies
path: /Users/justin/Dropbox/DJ MUSIC SORT/BEATJUNKIES
disk: 214.7 GB (46.1%) free

Choose mode:
 0: Single page
>0: Multiple pages
1

--- Page: 1 / 1 ---
Getting download links...
ERROR: <class 'AttributeError'>
'WebDriver' object has no attribute 'find_element_by_css_selector'
  File "/Users/justin/Documents/GitHub/recordpool-dl/src/RecordPoolDownloader.py", line 163, in <module>
    downloader.run_loop()

  File "/Users/justin/Documents/GitHub/recordpool-dl/src/RecordPoolDownloader.py", line 50, in run_loop
    self.multi_page_loop(number)

  File "/Users/justin/Documents/GitHub/recordpool-dl/src/RecordPoolDownloader.py", line 82, in multi_page_loop
    self.single_page_download()

  File "/Users/justin/Documents/GitHub/recordpool-dl/src/RecordPoolDownloader.py", line 98, in single_page_download
    tracks = self.pool.download_page(num_to_download)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/Users/justin/Documents/GitHub/recordpool-dl/src/RecordPool.py", line 84, in download_page
    tracks = self.get_tracks(num_to_download)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/Users/justin/Documents/GitHub/recordpool-dl/src/Beatjunkies.py", line 14, in get_tracks
    playlist = self.driver.find_element_by_css_selector(".widget.widget-beats.playlist")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

--------------------
Beatjunkies
Total files downloaded: 0 / 8.2 MB

[Beatjunkies.log](https://github.com/Esgrove/recordpool-dl/files/12114947/Beatjunkies.log)

**Comment by Esgrove** [comment]
Selenium update from version 3 to 4 changed some things, including how that `find_element` [method works](https://selenium-python.readthedocs.io/locating-elements.html). 

I have not used the Beatjunkies recordpool anymore since before the start of covid so probably the code is outdated anyway (assuming there has been some updates to the site).

I fixed the syntax for `find_element` now but since I only intended this tool for my own use and no longer use the Beatjunkies pool I won't be supporting it for other people. If you run into any more issues, you are on your own.

You might also want to change the download path, especially if you do not use Dropbox, since that is specific to my personal file organization...
